### ISSUE TITLE
Calliope: Motor A-Block and Motor B-Block work wrong: they move fast at speed 0 and slower at speed 100! 

### ISSUE THREAD

**Issue by kunkunuzzi** [OpenRoberta/robertalab#973]
Tested on multiple Calliopes: To reproduce this, you will need at least one dc-motor (and maybe a power source) and you will need to solder the motor pins to the calliope.
Please tell me, if you need further support, e.g. in testing.
I tried makecode and the abbozza platform too with the same Calliopes, none showed this bug. The latter one(abbozza) also translates blockly to C++ using "soundmotor" but it does not show the same bug, even if SoundMotor.cpp is nearly the same.

**Comment by bjost2s** [comment]
Could you please upload your testprogram as xml or screenshot here, so that we can reproduce this bug?
Thank you

**Comment by bjost2s** [comment]
Yes, thanks!
It takes some time for us to reproduce this (getting motors, calliopes and oscilloscope together).

**Comment by VinArt** [comment]
Hi @kunkunuzzi I have tested your code and noticed one interesting thing: when only 1 motor is set to a certain value, then it goes on a speed, that actually is more than 100% (and it does not matter whether it is 0 or 20, 40, 60 etc) the speed is constant, when I set both motors to some value (having 1 loop instead of 2) then they work as expected, so they are stopped at 0 and then turn faster, when the speed is increased. I am not sure whether it is a hardware issue, or the issue of the underlying library. I will have to look deeper into that. As for now, I can confirm that motors work in unexpected way, when in the 2 motor configuration the speed of only single motor is set to a certain value.
